### PR TITLE
Fixes for E_NOTICE in mod/item.php

### PR DESCRIPTION
**Pull request by Quix0r**
This PR fixes 2 E_NOTICE messages when `error_reporting` is set to `E_ALL`.

### CODE REVIEW

File: mod/item.php [hunk 1]
```diff
@@ -158,29 +158,32 @@ function item_post(App $a) {
...
+
 	if (!DBM::is_result($user) && !$parent) {
 		return;
 	}
 
 	$categories = '';
+	$postopts = '';
 
-	if ($orig_post) {
+	if (is_array($orig_post)) {
```
Comment (MrPetovan) [inline mod/item.php]: You could use `!empty()` instead, being an array wouldn't ensure the following array keys to be set.

Comment (Quix0r) [reply mod/item.php]: The initialization happens with `null`. Yeah, paradigms ... ;-) Okay, I change it.

File: src/Model/Item.php [hunk 2]
```diff
@@ -72,7 +72,7 @@ class Item extends BaseObject
 			'title', 'content-warning', 'body', 'location', 'coord', 'app',
 			'rendered-hash', 'rendered-html', 'object-type', 'object', 'target-type', 'target',
 			'author-id', 'author-link', 'author-name', 'author-avatar',
-			'owner-id', 'owner-link', 'owner-name', 'owner-avatar'];
+			'owner-id', 'owner-link', 'owner-name', 'owner-avatar', 'emailcc'];
```
Comment (annando) [inline src/Model/Item.php]: There is no "emailcc" field in the item table. What is this change supposed to do?

Comment (Quix0r) [reply src/Model/Item.php]: I have an E_NOTICE here in `mod/item.php`. I think @MrPetovan was not aware of it. So what should I do instead?

Comment (MrPetovan) [reply src/Model/Item.php]: Fix `mod/item.php` instead?

Comment (Quix0r) [reply src/Model/Item.php]: I was trying to find the root cause of the bug. :-) And I thought that the array `$orig_post` returned by `Item::selectFirst()` may miss that element.

**Comment by Quix0r** [comment]
So revert this merge?

**Comment by MrPetovan** [comment]
Just the commit with the `emailcc` add, but you can add it in a new branch with the fix to `mod/item.php`.

**Comment by Quix0r** [comment]
This is the line that causes the `E_NOTICE`. So I started backtracing it, ending up where I made the change:

https://github.com/friendica/friendica/pull/5393/files#diff-c51b2494e78a1bd1a25a46c76be08ba1R195

**Comment by Quix0r** [comment]
Thank you for making this clear. :+1: I create now a follow-up PR of this.

**Comment by Quix0r** [comment]
Or can I append to this?

**Comment by MrPetovan** [comment]
You can commit to the same branch but it will be a different PR.

**Comment by Quix0r** [comment]
Ah, to much changed now in that branch. Maybe it accidentally took over some other commits? I re-do it now.

**Comment by MrPetovan** [comment]
Part of #5470

**Pull request merged**